src/hidpp20.h

```diff
@@ -916,6 +922,7 @@ struct hidpp20_profiles {
 	uint8_t sector_count;
 	uint16_t sector_size;
 	uint8_t active_profile_index;
+	uint8_t profile_format_id;
```

Comment:
Following previous suggestion, use the above enum:
```suggestion
	enum hidpp20_onboard_profiles_profile_type profile_format_id;
```

Reply:
Actually, since, if I understand right, this is read from the mouse, there may be a value we don't yet know of, and so this won't necessarily fit an enumeration, so using an int should be better.

Reply:
Yes, just like this is the first device with a 0x05 profile format, there might be new devices with other profile format in the future.